apps/openmw/mwmechanics/aipackage.hpp

```diff
@@ -104,6 +104,9 @@ namespace MWMechanics
 
             virtual osg::Vec3f getDestination() { return osg::Vec3f(0, 0, 0); }
 
+            // Return true if any loaded actor with this AI package must be active.
+            virtual bool alwaysActive() { return false; }
```

Comment:
I suppose you missed the `const` modifier here, what leads to warnings.

Reply:
Huh, how could I miss it. Fixing now.